examples/vsphere-machinedeployment.yaml

```diff
@@ -37,20 +37,25 @@ spec:
             - "<< YOUR_PUBLIC_KEY >>"
           cloudProvider: "vsphere"
           cloudProviderSpec:
-            templateVMName: ubuntu-template
             # Can also be set via the env var 'VSPHERE_USERNAME' on the machine-controller
             username: '<< VSPHERE_USERNAME >>'
             # Can also be set via the env var 'VSPHERE_ADDRESS' on the machine-controller
             # example: 'https://your-vcenter:8443'. '/sdk' gets appended automatically
             vsphereURL: '<< VSPHERE_ADDRESS >>'
-            datacenter: 'Datacenter'
             # Can also be set via the env var 'VSPHERE_PASSWORD' on the machine-controller
             password:
               secretKeyRef:
                 namespace: kube-system
                 name: machine-controller-vsphere
                 key: password
-            cluster: "test-cluster"
+            datacenter: datacenter1
+            templateVMName: ubuntu-template
+            # Optional
+            templateNetName: ''
+            vmNetName: network1
+            # Optional
+            folder: folder1
```

Comment:
Are you sure, does it work when this is unset?

Reply:
Quite sure, in kubeone example I didn't used it and it's all good: https://github.com/kubermatic/kubeone/blob/master/examples/terraform/vsphere/outputs.tf#L57

Reply:
done